source/_UIAHandler.py

```diff
@@ -37,7 +37,7 @@
 import UIAUtils
 from comInterfaces import UIAutomationClient as UIA
 # F403: unable to detect undefined names
-from comInterfaces .UIAutomationClient import *  # noqa:  F403
+from comInterfaces.UIAutomationClient import *  # noqa:  F403

```

Comment:
Why this is in the diff?

Reply:
There was a space after comInterfaces which looks strange. Apparently python doesn't care, but I would like to fix it anyway.